ProjectSettings/TagManager.asset

```diff
@@ -4,7 +4,7 @@
 TagManager:
   serializedVersion: 2
   tags:
-  - Enemy
```

Comment:
Didn't we need the Enemy tag?
`PlayerManager.cs`

![image](https://user-images.githubusercontent.com/11907795/75307995-3680e680-581b-11ea-86df-89f11f4d97fe.png)

Reply:
Yeah I need to add that back, good catch.